docs/hibernating-clusters.md

```diff
@@ -0,0 +1,121 @@
+# Hibernating Clusters
+
+## Overview
+
+Starting with OpenShift version 4.4.8, clusters can be stopped and started by simply shutting
+down their machines and starting them back up. The only additional requirement to bring a cluster
+back up from a stopped state is that CSRs be approved for the cluster's nodes when certificates
+have expired since the cluster was active.
```

Comment:
"This most commonly happens if the cluster is hibernated in the first 24 hours before initial cert rotation takes place. After this phase, certs will be valid for 30 days."